.travis.yml

```diff
@@ -34,6 +34,10 @@ jobs:
       php: 7.2
       script: vendor/bin/infection
 
+    - stage: Verify BC Breaks
+      php: 7.2
+      script: bin/roave-backward-compatibility-check --from=master --to=HEAD
```

Comment:
Note: this will fail for now, as this patch does indeed lead to BC breaks (namespace changes)